Client/Frontend/Browser/Playlist/PlaylistToast.swift

```diff
@@ -0,0 +1,201 @@
+                $0.layer.borderWidth = ButtonToastUX.toastButtonBorderWidth
+                $0.layer.borderColor = UIColor.Photon.white100.cgColor
+                $0.setTitle(Strings.PlayList.toastAddToPlaylistOpenButton, for: [])
+                $0.setTitleColor(toastView.backgroundColor, for: .highlighted)
+                $0.titleLabel?.font = SimpleToastUX.toastFont
+                $0.titleLabel?.numberOfLines = 1
+                $0.titleLabel?.lineBreakMode = .byClipping
+                $0.titleLabel?.adjustsFontSizeToFitWidth = true
+                $0.titleLabel?.minimumScaleFactor = 0.1
+                
+                $0.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(buttonPressed)))
+            }
+
+            button.snp.makeConstraints { (make) in
+                make.width.equalTo(button.titleLabel!.intrinsicContentSize.width + 2 * ButtonToastUX.toastButtonPadding)
+            }
+
+            labelStackView.addArrangedSubview(label)
+            horizontalStackView.addArrangedSubview(labelStackView)
+            horizontalStackView.addArrangedSubview(button)
+
+            toastView.addSubview(horizontalStackView)
+
+            horizontalStackView.snp.makeConstraints { make in
+                make.centerX.equalTo(toastView)
+                make.centerY.equalTo(toastView)
+                make.width.equalTo(toastView.snp.width).offset(-2 * ButtonToastUX.toastPadding)
+            }
+
+            return toastView
+        }
+        
+        let horizontalStackView = UIStackView().then {
+            $0.axis = .horizontal
+            $0.alignment = .center
+            $0.spacing = ButtonToastUX.toastPadding
+        }
+        
+        let button = HighlightableButton().then {
+            $0.layer.cornerRadius = ButtonToastUX.toastButtonBorderRadius
+            $0.layer.masksToBounds = true
+            $0.backgroundColor = DownloadToastUX.toastBackgroundColor
+            $0.setTitleColor(toastView.backgroundColor, for: .highlighted)
+            $0.tintColor = UIColor.Photon.white100
+            $0.titleLabel?.font = UIFont.systemFont(ofSize: 15, weight: .medium)
+            $0.titleLabel?.numberOfLines = 1
+            $0.titleLabel?.lineBreakMode = .byClipping
+            $0.titleLabel?.adjustsFontSizeToFitWidth = true
+            $0.titleLabel?.minimumScaleFactor = 0.1
+            $0.contentHorizontalAlignment = .left
+            $0.contentEdgeInsets = UIEdgeInsets(top: 10.0, left: 10.0, bottom: 10.0, right: 20.0)
+            $0.titleEdgeInsets = UIEdgeInsets(top: 0.0, left: 10.0, bottom: 0.0, right: -10.0)
+            
+            $0.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(buttonPressed)))
+        }
+
+        button.snp.makeConstraints { make in
+            make.width.equalTo(button.titleLabel!.intrinsicContentSize.width + 2 * ButtonToastUX.toastButtonPadding)
```

Comment:
Unwrap `titleLabel`, i know it should never happen but we show the toast on popular video websites, we have to be super careful with crashes

Reply:
That particular code is copied from `ButtonToast.swift` for just the button.
I have unwrapped it anyway.